### PR TITLE
4.0.0

### PR DESCRIPTION
**Pull request by VincentLanglet**
Is anything more needed for the stable release @sonata-project/contributors ?

### CODE REVIEW

**Review by jordisala1991 — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,17 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.0.0](https://github.com/sonata-project/SonataClassificationBundle/compare/4.0.0-RC1...4.0.0) - 2022-01-01
+### Added
+- [[#778](https://github.com/sonata-project/SonataClassificationBundle/pull/778)] Added support for Symfony 6. ([@jordisala1991](https://github.com/jordisala1991))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#778](https://github.com/sonata-project/SonataClassificationBundle/pull/778)] Support for Symfony 6 ([@jordisala1991](https://github.com/jordisala1991))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,17 @@
...
+
+### Fixed
+- [[#786](https://github.com/sonata-project/SonataClassificationBundle/pull/786)] Fix admin controller configuration ([@core23](https://github.com/core23))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#786](https://github.com/sonata-project/SonataClassificationBundle/pull/786)] Admin controller configuration ([@core23](https://github.com/core23))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
...
+- [[#786](https://github.com/sonata-project/SonataClassificationBundle/pull/786)] Fix calling removed admin translator ([@core23](https://github.com/core23))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#786](https://github.com/sonata-project/SonataClassificationBundle/pull/786)] Calling removed admin translator ([@core23](https://github.com/core23))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,17 @@
...
+
+### Removed
+- [[#787](https://github.com/sonata-project/SonataClassificationBundle/pull/787)] Symfony deprecations ([@jordisala1991](https://github.com/jordisala1991))
```
Comment (phansys) [inline CHANGELOG.md:12]: IMO, this entry should be moved to the "Fixed" section.

**Review by jordisala1991 — DISMISSED** [review]

**Review by core23 — DISMISSED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,15 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.0.0](https://github.com/sonata-project/SonataClassificationBundle/compare/4.0.0-RC1...4.0.0) - 2022-01-01
+### Added
+- [[#778](https://github.com/sonata-project/SonataClassificationBundle/pull/778)] Support for Symfony 6. ([@jordisala1991](https://github.com/jordisala1991))
```
Comment (phansys) [inline CHANGELOG.md]:
```suggestion
- [[#778](https://github.com/sonata-project/SonataClassificationBundle/pull/778)] Support for Symfony 6 ([@jordisala1991](https://github.com/jordisala1991))
```

**Review by phansys — DISMISSED** [review]

**Pull request merged**